src/components/styles.scss

```diff
@@ -0,0 +1,173 @@
+$color1: #2c3e5d;
+$color2: #ccd6c9;
+$color3: #ebecf0;
+
+body {
+  overflow-x: unset;
+}
+
+.board-header {
+  position: fixed;
+  z-index: 10;
+  padding: 10px 24px;
+  background-color: #66ddaa;
+  min-width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  h1 {
+    color: white;
+    margin-bottom: 0;
+    font-size: 24px;
+  }
+}
+
+.board-scrollable {
+  padding-top: 50px;
+  height: 100vh;
+  min-width: 100%;
+  max-width: 100%;
+  overflow: auto;
+  background-color: $color2;
+}
+
+.board {
+  display: flex;
+  justify-content: flex-start;
+  min-height: 550px;
+  padding: 10px;
+  border-radius: 8px;
+  min-width: 100%;
+  border-radius: 8px;
+  .board_title {
+    display: flex;
+    justify-content: center;
+    font-size: 20px;
+    color: $color1;
+  }
+  .add-column-btn {
+    width: 80px;
+    height: 65px;
+    text-align: center;
+  }
+  .singout-btn {
+    justify-content: flex-end;
+  }
+}
+
+.list_item {
+  display: flex;
+  flex-direction: column;
+  min-width: 320px;
+  min-height: 100px;
+  padding: 10px 15px 10px 15px;
```

Comment:
you coud use shortcut:  padding: 10px 15px;

Reply:
fixed